devtools/conda-recipe/meta.yaml

```diff
@@ -22,7 +22,8 @@ requirements:
     - numpy >=1.11
     - scipy
     - cython
-    - netcdf4 ==1.3.1  # TODO: Fix this right after bugfix: "always return masked array by default, even if there are no masked values"
+    - netcdf4 >=1.4.2 # post-bugfix
```

Comment:
The new tests fail with a bunch of errors related to masked arrays. @Lnaden do you remember which was the issue number discussing the problem? We should check if this problem has been fixed in the latest release. Otherwise we can try to pin an old version of `libnetcdf` instead of a new one.

Reply:
See https://github.com/choderalab/openmmtools/pull/398#issuecomment-458981473